packages/core/src/components/date-picker-legacy/date-picker-legacy.tsx

```diff
@@ -157,6 +157,7 @@ export class DsoDatePickerLegacy implements ComponentInterface {
    * Defines a specific role attribute for the date picker input.
    */
   @Prop()
+  // eslint-disable-next-line @stencil-community/reserved-member-names
```

Comment:
Interessant. `role` gebruiken we als Input, maar ik lees nu dat het een "Global Attribute" is, ofwel: Dit attribuut zit op HTMLElement en "doet" daarom ook wat op browser niveau. Ik vraag mij af of we een toegankelijke werkvorm hebben.

Maar dit is de Legacy Date Picker, dus we gaan niets doen.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes